app/controllers/discover.js

```diff
@@ -126,17 +249,320 @@ export default Controller.extend(Analytics, {
         });
         return themeProvider;
     }),
-    _clearFilters() {
-        this.set('activeFilters', {
-            providers: [],
-            subjects: [],
+
+    searchUrl: computed(function() {
```

Comment:
This either doesn't need to be a computed property or should be dependent on `currentUser.sessionKey` (copied from ember-osf)